### ISSUE TITLE
Family Share 

### ISSUE THREAD

**Issue by iRng-ICEMAN** [FriendsOfGalaxy/galaxy-integration-steam#93]
**Off-Topic:** First of all, thanks for the work and the "recent" update which made the family share games playable from gog galaxy 2.0

**Problem:** It seems like that only the family share eligible games are shown in gog galaxy 2.0. For example, Post Scriptum is not enabled for family share and its not visible in gog galaxy 2.0.

Before the "recent" update, the games from family share would pop up in gog galaxy 2.0 but it would say "install", it wouldnt let you play -> so it was not working. Thats fixed now but the games which arent enabled for family share arent even visible in gog galaxy 2.0.

It would be great if this could be fixed somehow. This is essentially the last thing iam waiting for

**Own Idea:** Would it be possible to somehow manage 2 steam accounts in the background? Lets say the main account is logged in in gog galaxy 2.0 and the family share account is in the background, so the game would just launch through the family share account instead of using the family share feature itself.

**Comment by iRng-ICEMAN** [comment]
Any comments on this guys?

**Comment by Silphatos** [comment]
We haven't heard from FriendsofGalaxy for weeks, don't hold your breath :p

**Comment by iRng-ICEMAN** [comment]
> 
> 
> We haven't heard from FriendsofGalaxy for weeks, don't hold your breath :p

Mh oke :/ hope they are still around cause this is the only feature missing and it would be the cherry on top of the cake.

**Comment by FriendsOfGalaxy** [comment]
Hi @iRng-ICEMAN,

I'm not sure I understand a problem.

>  It seems like that only the family share eligible games are shown in gog galaxy 2.0.

Looks like intended behavior:
- games you own on your account are shown under "Steam" tab. 
- games that you don't know, but are coming from different account by Family Sharing are shown under "Family Sharing" under "Subscriptions" tab.

What does not work?

> games which arent enabled for family share arent even visible in gog galaxy 2.0.

So you want share games from different account even if Steam does not allow for this?

**Comment by iRng-ICEMAN** [comment]
> 
> 
> Hi @iRng-ICEMAN,
> 
> I'm not sure I understand a problem.
> 
> > It seems like that only the family share eligible games are shown in gog galaxy 2.0.
> 
> Looks like intended behavior:
> 
>     * games you own on your account are shown under "Steam" tab.
> 
>     * games that you don't know, but are coming from different account by Family Sharing are shown under "Family Sharing" under "Subscriptions" tab.
> 
> 
> What does not work?
> 
> > games which arent enabled for family share arent even visible in gog galaxy 2.0.
> 
> So you want share games from different account even if Steam does not allow for this?

Hey, nice to see you here : ) 

So, the initial issue was that i couldnt see the game "Post Scriptum" on GoG which was bought on a family shared account. But now, i assume that the game is generally not shown on GoG for other reasons ..

Reason for my assumption is that the game "Squad" (also doesnt support family share) is shown on GoG. When i look into my subscriptions tab (where my family shared steam account is shown) i can see Squad but not Post Scriptum. 

Now, lets come to the actual "problem" (feature request). Games which dont support family share (for example Squad) arent playable through the family shared tab. It doesnt say "play", instead it says "install" cause it doesnt recognise the game is already installed or whatever.

It would be the cherry on the cake if this could somehow be implemented. Maybe the option to log in 2 steam accounts into GoG Galaxy 2.0 at the same time?

**Comment by FriendsOfGalaxy** [comment]
> Reason for my assumption is that the game "Squad" (also doesnt support family share) is shown on GoG.

That's interesting. Apparently according to Steam protocols it is sharable (we just read a boolean flag). Anyway - nobody suffers if it isn't and we still show it.

> Games which dont support family share (for example Squad) arent playable through the family shared tab. It doesnt say "play", instead it says "install" cause it doesnt recognise the game is already installed or whatever.

@iRng-ICEMAN Share your logs (https://github.com/FriendsOfGalaxy/galaxy-integration-steam/wiki/Log-files) - it is hard to guess what happened. It should work out to the box - locally installed games are handled separately from the owned / subscription games.

**Comment by iRng-ICEMAN** [comment]
> That's interesting. Apparently according to Steam protocols it is sharable (we just read a boolean flag). Anyway - nobody suffers if it isn't and we still show it.
> 

Yeah exactly, that doesnt hurt and is actually the better way to do it.

> @iRng-ICEMAN Share your logs (https://github.com/FriendsOfGalaxy/galaxy-integration-steam/wiki/Log-files) - it is hard to guess what happened:) It should work out to the box - locally installed games are handled separately from the owned / subscription games. > 

I think i couldnt really explain my issue.

Squad is installed and is a subscription games (its from the family share account (steam)). Its not that i have linked a game shortcut to gog ... The game (squad) comes from my family shared account and is installed. When i switch my steam account to my fmaily share account, i can just start up the game from steam.

**Comment by FriendsOfGalaxy** [comment]
@iRng-ICEMAN share your logs or we can guess what happened in the code for entire eternity :) Or I can try to reproduce it on my computer, but currently I have no peer next to me who could share games on my machine.
